.github/workflows/e2e-test.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [main]
   pull_request:
-    types: [synchronize, opened, reopened, edited]
```

Comment:
I changed this, as the e2e tests would run whenever the main body or title was edited, which I don't think makes sense. I saw that `edited` is also triggered when the base branch changes, but I guess that is a less common scenario and in that case kodiak would merge anyway.